cmd/node/factory/structs.go

```diff
@@ -575,6 +580,45 @@ func ProcessComponentsFactory(args *processComponentsFactoryArgs) (*Process, err
 	}, nil
 }
 
+func indexGenesisAccounts(accountsAdapter state.AccountsAdapter, indexer indexer.Indexer, marshalizer marshal.Marshalizer) error {
+	rootHash, err := accountsAdapter.RootHash()
+	if err != nil {
+		return err
+	}
+
+	leaves, err := accountsAdapter.GetAllLeaves(rootHash)
+	if err != nil {
+		return err
+	}
+
+	genesisAccounts := make([]state.UserAccountHandler, 0)
+	for addressKey, userAccountsBytes := range leaves {
+		userAccount, errUnmarshal := unmarshalUserAccount([]byte(addressKey), userAccountsBytes, marshalizer)
+		if errUnmarshal != nil {
+			log.Warn("cannot unmarshal genesis user account. it may be a code leaf", "error", errUnmarshal)
```

Comment:
Since this will almost certain happen, why not switching this to Debug level?

Reply:
done